### PR TITLE
[release-3.6] health checks: tolerate ovs 2.9

### PR DESCRIPTION
**Pull request by sosiouxme**
backport of #7035

### CODE REVIEW

File: roles/openshift_health_checker/openshift_checks/ovs_version.py [hunk 1]
```diff
@@ -16,8 +16,8 @@ class OvsVersion(NotContainerizedMixin, OpenShiftCheck):
     tags = ["health"]
 
     openshift_to_ovs_version = {
-        "3.7": ["2.6", "2.7", "2.8"],
-        "3.6": ["2.6", "2.7", "2.8"],
+        "3.7": ["2.6", "2.7", "2.8", "2.9"],
+        "3.6": ["2.6", "2.7", "2.8", "2.9"],
```
Comment (Miciah) [inline roles/openshift_health_checker/openshift_checks/ovs_version.py]: Seems a little weird to have a mapping for "3.7" in the release-3.6 branch, but it shouldn't hurt anything.

**Comment by sosiouxme** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-ansible/7038/test_pull_request_openshift_ansible_extended_conformance_install_system_containers/4673/

```
+ sudo atomic pull --storage ostree docker:openshift/node:v3.6.1
Error response from daemon: reference does not exist
```

Gonna call that an infra flake.

File: roles/openshift_health_checker/openshift_checks/ovs_version.py (hunk 1 above)
Comment (sosiouxme) [reply roles/openshift_health_checker/openshift_checks/ovs_version.py]: That's what I get for cherry-picking

**Pull request merged**